test/integration/targets/win_psrepository/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+# This file is part of Ansible
+
+# Copyright: (c) 2018, Wojciech Sciesinski <wojciech[at]sciesinski[dot]net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: get facts
```

Comment:
Facts are really slow, just have a `raw` or `win_shell` that runs `$PSVersionTable.PSVersion.Major`